docs/release_notes.md

```diff
@@ -1,5 +1,39 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Bug Fixes
+
+- Filtered characters to get valid names for nodes and attributes since some special characters are not allowed in Maya for Linux. *AdonisFX-1033*
+- Fixed Python runtime error due to incorrect element selection on node creation. *AdonisFX-1031*
+- Fixed error messages not displaying the proper status of the AdnSimshape connections. *AdonisFX-1034*
+- Fixed exporter creating the AAD file when there is nothing to export or the only thing to export failed. *AdonisFX-1032*
+- Fixed manipulation of referenced nodes while using the AdonisFX paint tool. *AdonisFX-1147*
+- Fixed a bug that did not allow to add attachments on a AdnMuscle/AdnRibbonMuscle deformer created on referenced geometry. *AdonisFX-506*
+- Fixed a bug in the handling of the constraint weight arrays. *AdonisFX-1155*
+- Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on for some vertices in very specific scenarios. *AdonisFX-1165*
```

Comment:
I would not add the part of "very specific scenarios". Instead I would point where the change is affecting.

```suggestion
- Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialization. *AdonisFX-1165*
```

Reply:
Done.